litmus-portal/graphql-server/pkg/project/create.go

```diff
@@ -164,38 +161,11 @@ func getInvitation(ctx context.Context, member model.MemberInput) (dbSchema.Invi
 	return "", nil
 }
 
-//RemoveMember ...
-func RemoveMember(ctx context.Context, member model.MemberInput) (string, error) {
-	invitation, err := getInvitation(ctx, member)
-	if err != nil {
-		return "Unsuccessful", err
-	}
-	if invitation == dbSchema.AcceptedInvitation {
-		err = database.RemoveMember(ctx, member.ProjectID, member.UserName)
-		if err != nil {
-			return "Unsuccessful", err
-		}
-	} else {
-		return "Unsuccessful", errors.New("User is not a member of the project")
-	}
-	return "Member successfully removed", nil
-}
-
-//CancelInvite ...
-func CancelInvite(ctx context.Context, member model.MemberInput) (string, error) {
-	invitation, err := getInvitation(ctx, member)
+//RemoveInvitation ...
+func RemoveInvitation(ctx context.Context, member model.MemberInput) (string, error) {
+	err := database.RemoveInvitation(ctx, member.ProjectID, member.UserName)
 	if err != nil {
 		return "Unsuccessful", err
 	}
-	if invitation == dbSchema.AcceptedInvitation {
-		return "Unsuccessful", errors.New("Invitation already Accepted")
-	} else if invitation == dbSchema.PendingInvitation {
-		err = database.CancelInvite(ctx, member.ProjectID, member.UserName)
-		if err != nil {
-			return "Unsuccessful", err
-		}
-	} else if invitation == dbSchema.DeclinedInvitation {
-		return "Unsuccessful", errors.New("Invitation already Rejected")
-	}
-	return "Invitation successfully cancelled", nil
+	return "Successful", nil
```

Comment:
We should have this available only for `AcceptedInvitation` or `PendingInvitation`. If the invitation is `DeclinedInvitation` , it should return an error

Reply:
@Sanjay1611 To solve these error message issues we will have to create two separate functions (cancelInvite and removeMember).